Driverless AI/ES/automatic-ml-intro-test-drive-tutorial/automatic-ml-intro-test-drive-tutorial.md

```diff
@@ -87,27 +87,27 @@ La interfaz del usuario (UI) de Driverless AI es muy fácil de navegar. Las sigu
 
-En el Machine Learning, las variables de entrada son llamadas **características(features)** y las variables de salida **etiquetas(labels)**. Las etiquetas, en este caso, son las que intentamos predecir. El objetivo es tomar las entradas/características y usarlas para llegar a predicciones sobre datos nunca antes vistos. En regresión lineal, las características son las variables x, y las etiquetas son las variables y. 
+En Aprendizaje de Maquina, las variables de entrada son llamadas **características(features)** y las variables de salida **etiquetas(labels)**. Las etiquetas, en este caso, son las que intentamos predecir. El objetivo es tomar las entradas/características y usarlas para llegar a predicciones sobre datos nunca antes vistos. En regresión lineal, las características son las variables x, y las etiquetas son las variables y. 
 
-Un modelo de Machine Learning define la relación entre características y etiquetas. Cuando los modelos son entrenados, puedes entrenar el modelo alimentándolo con ejemplos. Los ejemplos son una instancia particular de datos. Puedes tener dos tipos de ejemplos: etiquetados y no-etiquetados. Los ejemplos etiquetados son esos donde se conoce el valor de las variables x, y (características, etiquetas). Los ejemplos no-etiquetados son esos donde conocemos el valor de la variable x, pero no sabemos que valor tiene la variable y(características,?)[1]. Tu conjunto de datos son como un ejemplo; las columnas que se usarán para el entrenamiento son las características; las filas son las instancias de esas características. Las columnas que quieres predecir son las etiquetas.
+Un modelo de Aprendizaje de Maquina define la relación entre características y etiquetas. Cuando los modelos son entrenados, puedes entrenar el modelo alimentándolo con ejemplos. Los ejemplos son una instancia particular de datos. Puedes tener dos tipos de ejemplos: etiquetados y no-etiquetados. Los ejemplos etiquetados son esos donde se conoce el valor de las variables x, y (características, etiquetas). Los ejemplos no-etiquetados son esos donde conocemos el valor de la variable x, pero no sabemos que valor tiene la variable y(características,?)[1]. Tu conjunto de datos son como un ejemplo; las columnas que se usarán para el entrenamiento son las características; las filas son las instancias de esas características. Las columnas que quieres predecir son las etiquetas.
 
 El aprendizaje supervisado toma los ejemplos etiquetados y permite a un modelo que esta siendo entrenado aprender la relación entre características y etiquetas. El modelo entrenado es entonces probado con datos no-etiquetados, y eso permite predecir el valor de y(etiqueta) para los datos no-etiquetados. Probar un modelo entrenado con datos no-etiquetados se le llama entrenamiento no supervisado [1]. Note que H20 Driverless AI crea modelos con ejemplos etiquetados.
 
 ### Preparación de datos 
 
```

Comment:
Testing for review

Reply:
Just to make sure this was just a test, right? No review is required?